app/main.py

```diff
@@ -73,6 +73,17 @@ async def lifespan(app_: FastAPI):
     """Run startup and shutdown events."""
     logger.info("Starting up...")
     run_migrations(engine)
+    scheduler = BackgroundScheduler()
```

Comment:
I'm fairly certain this won't save us as if another instance is bought up this startup lifespan event will still be triggered!

Reply:
Does that mean we run migrations every time the scaling forks a new node? For what its worth Im not super confident this will save us either but didn't have time to switch to something better this week, If it doesnt work on Monday my plan would be to just remove it and hopefully will be able to replace it in prefect by the next one

Reply:
From my understanding - yes; we run alembic upgrade head on startup (this is pre the db client work) and thus if we are at the correct revision there would just be no updates. 

I'll approve so we can give this a go and me be proven wrong haha!

Reply:
🤞